src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -0,0 +1,141 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.errorconsole;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonBar;
+import javafx.scene.control.DialogPane;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.text.Text;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+
+import net.sf.jabref.gui.FXAlert;
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.error.MessagePriority;
+import net.sf.jabref.logic.error.ObservableMessageWithPriority;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.logging.ObservableMessages;
+
+import com.airhacks.afterburner.views.FXMLView;
+
+public class ErrorConsoleView extends FXMLView {
+
+    private final ErrorConsoleViewModel errorViewModel = new ErrorConsoleViewModel();
+
+    @FXML
+    private Button closeButton;
+    @FXML
+    private Button copyLogButton;
+    @FXML
+    private Button createIssueButton;
+    @FXML
+    private ListView<ObservableMessageWithPriority> allMessage;
+    @FXML
+    private Label descriptionLabel;
+
+    public ErrorConsoleView() {
+        super();
+        bundle = Localization.getMessages();
+    }
+
+    public void show() {
+        FXAlert errorConsole = new FXAlert(AlertType.ERROR, Localization.lang("Developer information"), false);
+        DialogPane pane = (DialogPane) this.getView();
+        errorConsole.setDialogPane(pane);
+        errorConsole.setResizable(true);
+        errorConsole.show();
+    }
+
+    @FXML
+    private void initialize() {
+        ButtonBar.setButtonData(copyLogButton, ButtonBar.ButtonData.LEFT);
+        ButtonBar.setButtonData(createIssueButton, ButtonBar.ButtonData.LEFT);
+
+        ObservableList<ObservableMessageWithPriority> masterData = ObservableMessages.INSTANCE.messagesPropety();
+        listViewStyle();
+        allMessage.setItems(masterData);
+
+        //set console icon instead the error icon and insert a description
+        Text graphic = new Text(IconTheme.JabRefIcon.CONSOLE.getCode());
```

Comment:
In one of the recent FX-PRs (I hope it is already merged), a new method in `JabRefIcon` was introduced which gives you directly the `Text` control with the right style class.